tests/components/yeelight/test_config_flow.py

```diff
@@ -203,7 +203,9 @@ async def test_manual(hass: HomeAssistant):
         result4 = await hass.config_entries.flow.async_configure(
             result["flow_id"], {CONF_HOST: IP_ADDRESS}
         )
+        await hass.async_block_till_done()
     assert result4["type"] == "create_entry"
+    assert result4["title"] == IP_ADDRESS
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/manual/foreign-sub/home-assistant/pullRequest?prid=6372789)